### PR TITLE
refactor(condo): HOTFIX remove old analytics service

### PR DESCRIPTION
**Pull request by toplenboren**

### CODE REVIEW

File: apps/condo/migrations/20240825174532-0418_remove_externalreport_createdby_and_more.js
Comment (Alllex202) [inline apps/condo/migrations/20240825174532-0418_remove_externalreport_createdby_and_more.js:1]: you should comment out all commands and add it to `apps/condo/bin/fix-database-migration-state.js`

File: apps/condo/domains/analytics/utils/clientSchema/index.ts
Comment (Alllex202) [inline apps/condo/domains/analytics/utils/clientSchema/index.ts:1]: What about server and test utils?

**Review by Alllex202 — COMMENTED** [review]
Also should mark Organization.canReadExternalReports` as deprecated - at some point in future the field will need to be removed

**Comment by Alllex202** [comment]
Let's add the task number :)

File: apps/condo/migrations/20240825174532-0418_remove_externalreport_createdby_and_more.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Delete model externalreport
+--
+DROP TABLE "ExternalReport" CASCADE;
```
Comment (pahaz) [inline apps/condo/migrations/20240825174532-0418_remove_externalreport_createdby_and_more.js:10]: Move it to fix migration file according to ADR

File: apps/condo/migrations/20240825174532-0418_remove_externalreport_createdby_and_more.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    `)
+}
+
+exports.down = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Delete model externalreporthistoryrecord
+--
+CREATE TABLE "ExternalReportHistoryRecord" ("type" text NULL, "title" text NULL, "description" text NULL, "meta" jsonb NULL, "organization" uuid NULL, "isHidden" boolean NULL, "id" uuid NOT NULL PRIMARY KEY, "v" integer NULL, "createdAt" timestamp with time zone NULL, "updatedAt" timestamp with time zone NULL, "createdBy" uuid NULL, "updatedBy" uuid NULL, "deletedAt" timestamp with time zone NULL, "newId" jsonb NULL, "dv" integer NULL, "sender" jsonb NULL, "history_date" timestamp with time zone NOT NULL, "history_action" varchar(50) NOT NULL, "history_id" uuid NOT NULL);
```
Comment (pahaz) [inline apps/condo/migrations/20240825174532-0418_remove_externalreport_createdby_and_more.js:26]: if not exists

**Review by pahaz — APPROVED** [review]

**Review by Alllex202 — APPROVED** [review]

**Pull request merged**